Content.Server/Temperature/Systems/TemperatureSystem.cs

```diff
@@ -139,42 +139,66 @@ private void OnRejuvenate(EntityUid uid, TemperatureComponent comp, RejuvenateEv
 
     private void ServerAlert(EntityUid uid, AlertsComponent status, OnTemperatureChangeEvent args)
     {
-        switch (args.CurrentTemperature)
+        if (TryComp<TemperatureComponent>(uid, out var temperature))
         {
-            // Cold strong.
-            case <= 260:
-                _alertsSystem.ShowAlert(uid, AlertType.Cold, 3);
-                break;
+            var cold3 = temperature.ColdDamageThreshold;
```

Comment:
Instead of hard coding the difference between each step, then checking later to see if the ranges overlap, why isn't this just divide the difference between HeatDamageThreshold and ColdDamageThreshold into several ranges?

Reply:
I wanted to keep the width of the individual warning bands absolute, not relative. That seemed more "consistent" to me

Reply:
I came around to see that dynamic alert temperatures are better so I vultured your code, thank you